portfolio/src/main/java/com/google/sps/servlets/ListCommentServlet.java

```diff
@@ -0,0 +1,96 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.*;
+import com.google.gson.Gson;
+import com.google.sps.data.Comment;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.google.sps.utils.Constants.COMMENT_KEY;
+import static com.google.sps.utils.Constants.COMMENT_MAXNUMBER;
+import static com.google.sps.utils.Constants.COMMENT_TIMESTAMP;
+
+
+/** Servlet that handles posting comment content. */
+@WebServlet("/list-comment")
+public class ListCommentServlet extends CommentServlet {
+    private int maxNumberOfComments;
+
+    @Override
+    public void init() {
+        this.maxNumberOfComments = 0; // default value
+    }
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        // Loads the comment from Datastore
```

Comment:
If we provide pagination, in the get, we can read param page index and page size or assign a default value if it's missing. In the return value, we'll provide a page object with page index, page size, total count so that the caller know which page they're getting and in total how many pages.

Reply:
As discussed offline, since pagination is not covered in the tutorial yet, i will just get the maxNumberOfComments variable in doGet() function instead of doPost() function